docs/components/modeler/bpmn/escalation-events/escalation-events.md

```diff
@@ -0,0 +1,73 @@
+This will throw an escalation to a higher flow scope. The escalation is caught by the `Catch` event. As escalation events
+are non-critical, the outgoing sequence flows of `Throw` and `Catch` are both taken.
+
+## Defining an escalation
+
+In BPMN, an `escalation event` references an `escalation`. Escalations can be referenced by one or more escalation events.
+
+An escalation must define an `escalationCode`. The value of this `escalationCode` is used to determine which catch event
+can catch the thrown escalation.
+
+For throwing escalation events it is possible to define the `escalationCode` as an `expression`. When the event is reached the expression gets evaluated.
+An escalation with the result of this expression is thrown. If no expression is used the statically defined `escalationCode` is used.
+
+For catching escalation events it is not possible to use an `expression`. The `escalationCode` must always be a static value.
+Alternatively, the `escalationCode` can be left empty. A catch event with an empty `escalationCode` will catch **all** thrown escalations.
+
+## Throwing the escalation
+
+An escalation can be thrown by an escalation end event, or by an intermediate escalation throw event. Escalation events
+are non-critical. This means that if the throwing event has any outgoing sequence flows, they will be taken.
+
+## Catching the escalation
+
+An escalation can be caught using a boundary event, or using an event subprocess. It is caught by one catch event at most.
+This will be the catch event in the nearest flow scope.
+
+It is not possible to define multiple escalation catch events with the same `escalationCode` in a single scope.
+The deployment gets rejected in this case. However, it is possible to define both an escalation catch event **with** an
+`escalationCode` and one **without** an `escalationCode` in the same scope. When this happens the escalation catch event
+that matches the `escalationCode` is prioritized.
+
+If there are no escalation catch events that match the `escalationCode` the escalation will not be caught. Unlike with
+[error events](../error-events/error-events.md) no incident gets raised. The process will continue without escalating.
+
+Even though escalations are non-critical, it is still possible make escalation catch events interrupting. This will
+behave the same as other interrupting events. The catch event will terminate the scope it is attached to. In this case
+the outgoing sequence flows of the throwing escalation event are not taken.
+
+## Additional resources
+
+### XML representation
+
+An intermediate escalation event with expression:
+
+```xml
+<bpmn:intermediateThrowEvent id="StartEvent_1">
+    <bpmn:escalationEventDefinition id="EscalationEventDefinition_0sdm9od" escalationRef="Escalation_2alpsjo" />
+</bpmn:intermediateThrowEvent>
+
+<bpmn:escalation id="Escalation_2alpsjo" name="Escalation_2alpsjo" escalationCode="=escalationCode" />
+```
+
+An escalation boundary event:
```

Comment:
:wrench: Same as above since we mention catchevents in the sections above, it would be good that we keep up the same terminology here:

```suggestion
An escalation boundary catch event:
```